app/views/profile/index.html.slim

```diff
@@ -1,5 +1,4 @@
-- content_for(:nav) do
-  = render 'shared/nav_auth'
+= shared_nav(greeting: @view_model.header_personalization)
```

Comment:
yeah you are right! how about testing this method via testing the method in  `app/view_models/profile_index.rb`-- that's where the logic lives, anyway/